lib/standalone.js

```diff
@@ -170,7 +170,7 @@ module.exports = function(
 		fileList = [fileList];
 	}
 
-	if (!options.disableDefaultIgnores) {
+	if (!options.disableDefaultIgnores && !options.noglob) {
```

Comment:
In this case, `ALWAYS_IGNORED_GLOBS` is not working as expected with `noglob`, it is not good idea